samples/stock/plotoptions/series-datagrouping-approximation/demo.js

```diff
@@ -33,7 +33,7 @@ Highcharts.stockChart('container', {
         data: ADBE,
         dataGrouping: {
             approximation: function () {
-                var start = this.cropStart + this.dataGroupInfo.start;
+                var start = this.dataGroupInfo.start;
```

Comment:
From my experience, deleting some code (instead of changing it or adding new cases) is dangerous. Did you think about why the `this.cropStart` was there initially? Did you remove it consciously?

Maybe there is a use case when `this.cropStart` is actually necessary. For example:
```js
var start;

if (somethingThatGivesUsCropStart) {
    start = this.cropStart + this.dataGroupInfo.start;
} else { // if cropStart is unnecessary
    start = this.dataGroupInfo.start;
}
```

I don't know, I'm just asking whether you're sure what you're doing.

Reply:
There is no need to longer add `this.cropStart` since this is already included in the `this.dataGroupInfo.start`
Related [commit](https://github.com/highcharts/highcharts/commit/2d0d702c6493aad166ced3d6bb3df07771fa5c54).